lib/renaissance_web/router.ex

```diff
@@ -17,6 +17,8 @@ defmodule RenaissanceWeb.Router do
     pipe_through :browser
 
     get "/", PageController, :index
+    get "/register", RegisterController, :index
```

Comment:
`:index` is a bit strange here. It's not an index. I think this is really your `:new` action. and `post "/register"` is probably `:create` or `:register`. The convention is:

`get "/things"` => `:index` (A list view of things)
`get "/things/new"` => `:new` (A form for a new thing)
`post "/things` => `:create` (Create a new thing from the form mounted at "/things/new")